docs/zkapps/o1js/interact-with-mina.md

```diff
@@ -46,19 +67,23 @@ const tx = await Mina.transaction(sender, () => {
 });
 ```
 
-In this example, the transaction consists of calling a single `SmartContract` method, called `myMethod`. You can inspect the transaction yourself by printing it out as JSON:
+In this example, the transaction calls a single `SmartContract` method called `myMethod`. You can inspect the transaction by printing the JSON data to the console:
 
 ```ts
 console.log(tx.toJSON());
 ```
 
-If you try this, you see a massive JSON object with lots of fields, most of which are set to their default value. There's also a way to pretty-print transactions in a more human-readable, condensed format:
+This command outputs a massive JSON object with many fields, most of which are set to their default value. 
+
+### Transaction output
+
+Instead, it's better to output transactions in a more human-readable, condensed format to print only the fields with actual content:
```

Comment:
there is a lot of legacy "if you do this, it fails" content in the docs... should we remove in this case?

Reply:
I hear ya... and I understand. For context, the other docs content that I considered legacy was the earlier versions of tutorial steps that said "run this command, but it will break" which are now rewritten